apps-rendering/package-lock.json

```diff
@@ -3376,7 +3376,7 @@
         "ms": {
```

Comment:
You may want to drop the changes to this file?

Reply:
Agreed!